src/EA.Weee.Web/Areas/Admin/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -102,10 +107,93 @@ public async Task<ActionResult> ViewEvidenceNoteTransfer(Guid evidenceNoteId)
                 var model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(result.TransferredOrganisationData.Id,
                    result, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]));
 
+                model.CanVoid = IsInternalAdminUser();
+
                 return View(model);
             }
         }
 
+        [HttpGet]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        public async Task<ActionResult> VoidTransferNote(Guid transferEvidenceNoteId)
+        {
+            SetBreadcrumb(BreadCrumbConstant.ManageEvidenceNotesAdmin);
+
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteTransfersForInternalUserRequest(transferEvidenceNoteId);
+
+                var transferNoteData = await client.SendAsync(User.GetAccessToken(), request);
+
+                if (transferNoteData.Type == NoteType.Transfer && transferNoteData.Status == NoteStatus.Approved)
+                {
+                    var model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(transferNoteData.TransferredOrganisationData.Id,
+                       transferNoteData, null));
+
+                    //model.CanVoid = IsInternalAdminUser();
+
+                    return View("VoidTransferNote", model);
+                }
+
+                return new HttpStatusCodeResult(HttpStatusCode.Forbidden, "This note is not a transfer note or has not been approved.");
+            }
+        }
+
+        [HttpPost]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> VoidTransferNote(ViewTransferNoteViewModel model)
+        {
+            SetBreadcrumb(BreadCrumbConstant.ManageEvidenceNotesAdmin);
+
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteTransfersForInternalUserRequest(model.EvidenceNoteId);
+
+                var result = await client.SendAsync(User.GetAccessToken(), new VoidTransferNoteRequest(model.EvidenceNoteId, model.VoidedReason));
+
+                var transferNoteData = await client.SendAsync(User.GetAccessToken(), request);
+
+                if (transferNoteData.Type == NoteType.Transfer && transferNoteData.Status == NoteStatus.Void)
+                {
+                    model.CanVoid = IsInternalAdminUser();
```

Comment:
@GuidoWorkDev We don't need to set this here? Rather than doing
return RedirectToAction("ViewTransferNote", "ManageEvidenceNotes", new { transferNoteId = model.EvidenceNoteId });

Do

return RedirectToAction("ViewEvidenceNoteTransfer", "ManageEvidenceNotes", new { transferNoteId = model.EvidenceNoteId });

As we already have an action that does the same as ViewTransferNote, unless there is a reason for this new view action...